Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Motion/MotionDataBuilder.cpp

```diff
@@ -347,11 +347,12 @@ namespace EMotionFX
                         // For additive motion, we stores the relative transform.
                         boneTransform = sampleFrameTransformInverse * boneTransform;
                     }
-                    
+
                     SceneAPIMatrixType boneTransformNoScale(boneTransform);
-                    const AZ::Vector3 position = coordSysConverter.ConvertVector3(boneTransform.GetTranslation());
-                    const AZ::Quaternion rotation = coordSysConverter.ConvertQuaternion(AZ::Quaternion::CreateFromMatrix3x4(boneTransformNoScale));
-                    const AZ::Vector3 scale = coordSysConverter.ConvertScale(boneTransformNoScale.ExtractScale());
+                    const AZ::Vector3 scale = boneTransformNoScale.ExtractScale();
+                    const AZ::Transform convertedTransform = AZ::Transform::CreateFromMatrix3x4(coordSysConverter.ConvertMatrix3x4(boneTransformNoScale));
```

Comment:
suggestion: We could do this as a follow-up or leave for now, but it would be great to include this `ExtractScale` call in a wrapper function so ensure it happens (and we can then test the fail before/after). I sometimes like to include the new test that fails with the old code, and then passes with the new code in the description🙂.